app/serializers/concerns/cached_serializer.rb

```diff
@@ -0,0 +1,17 @@
+module CachedSerializer
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def as_json(model, context)
+      if Panoptes.flipper["cached_serializer"].enabled?
+        cache_key = "#{model.class.to_s}/#{model.id}/#{model.updated_at.to_i}/context-#{Digest::MD5.hexdigest(context.to_json)}"
```

Comment:
No real reason, just came out of adding that digest later on in the development. Though once we use some external cache store we might enjoy being able to see which objects have caches, which would only really be possible with human readable keys.